### PR TITLE
Scmf create reviews

### PR DESCRIPTION
**Pull request by sallypikachu**
Merged current master branch and fixed conflicts
Created reviews
Updated all activity name switch to venue

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @venue = Venue.find(params[:venue_id])
+    @review = Review.new
+    @quantity_collection = [[1,1], [2,2], [3,3], [4,4], [5,5]]
+  end
+
+  def create
+    @quantity_collection = [[1,1], [2,2], [3,3], [4,4], [5,5]]
+    @venue = Venue.find(params[:venue_id])
+    @review = Review.new(review_params)
+    @review.venue = @venue
```
Comment (davidrf) [inline app/controllers/reviews_controller.rb]: change lines 12-13 to `@review = Venue.reviews.new(review_params)`

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @review.user_id = 1
```
Comment (davidrf) [inline app/controllers/reviews_controller.rb]: You want to set it to the use who is currently signed in, so change this line to `@review.user = current_user`

File: spec/features/03_user_sees_reviews_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# [] Visiting the `/venues` path should contain a list of restaurants.
+# [] Visiting the `/venues/10 path should show the details for a venue with the ID of 10.
+# [] Visiting the root path should display a list of all venues.
+
+feature "user sees a list of reviews for the given acitivty" do
+  before(:each) do
+    @venue = FactoryGirl.create(:venue)
+    @a_review = FactoryGirl.create(:review, venue_id: @venue.id)
+    @b_review = FactoryGirl.create(:review, venue_id: @venue.id)
+  end
```
Comment (davidrf) [inline spec/features/03_user_sees_reviews_spec.rb]: Use `let` for all of these `let!(:venue) { FactoryGirl.create(:venue) }`

File: spec/features/03_user_sees_reviews_spec.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    click_button "Add review"
+
+    expect(page).to have_content("Cool")
+    expect(page).to have_content("4")
+
+  end
+
+  scenario "see average rating" do
+
+  end
```
Comment (davidrf) [inline spec/features/03_user_sees_reviews_spec.rb]: Delete this empty scenario block

File: db/schema.rb [hunk 5]
```diff
@@ -11,11 +11,21 @@
...
+ActiveRecord::Schema.define(version: 20160106204250) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "reviews", force: :cascade do |t|
+    t.integer  "rating",      null: false
+    t.text     "review_body"
+    t.integer  "user_id",     null: false
+    t.integer  "delete_ind"
```
Comment (davidrf) [inline db/schema.rb]: Remove this attribute

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (sallypikachu) [inline app/controllers/reviews_controller.rb]: This doesn't work!

File: spec/features/03_user_sees_reviews_spec.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+feature "user sees a list of reviews for the given acitivty" do
+  let!(:venue) { FactoryGirl.create(:venue) }
+  let!(:review) { FactoryGirl.create(:review, venue_id: venue.id) }
+
+  scenario "clicks link and is taken to show page for given venue" do
+    visit venues_path
+    click_link venue.venue_name
+
+    expect(page).to have_content venue.venue_name
+    expect(page).to have_content venue.description
```
Comment (davidrf) [inline spec/features/03_user_sees_reviews_spec.rb]: Delete lines 16-17, the details of the venue are part of another test.

File: spec/features/03_user_sees_reviews_spec.rb [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    expect(page).to have_content review.review_body
+
+    click_link "All Venues"
```
Comment (davidrf) [inline spec/features/03_user_sees_reviews_spec.rb]: Remove this line, I don't see why you're clicking this link at the end.

**Pull request merged**